schemata/share.yaml

```diff
@@ -52,3 +52,6 @@ properties:
         example: Bergie Today
     required: [url]
 required: [url]
+dependencies:
```

Comment:
Me neither. But trying to get the validation to be as close as possible to reality made me peruse the JSON schema docs and find bunch of goodies.

Ideally we should get these to stage where schemas can be trusted for all other validation except side effects (UUID references matching database etc). But that will probably take a few iterations